tutorials/example_keras_qat.py

```diff
@@ -0,0 +1,130 @@
+                            enable_weights_quantization=False,
+                            enable_activation_quantization=False))
+
+    tpc = tp.TargetPlatformCapabilities(tp_model)
+    with tpc:
+        # No need to quantize Flatten and Dropout layers
+        tp.OperationsSetToLayers("NoQuantization", [layers.Flatten,
+                                                    layers.Dropout])
+
+    return tpc
+
+
+def get_model(_num_classes, _input_shape):
+    _input = layers.Input(shape=_input_shape)
+    x = layers.Conv2D(16, 3, strides=2, padding='same', activation='relu')(_input)
+    x = layers.Conv2D(32, 3, strides=2, padding='same', activation='relu')(x)
+    x = layers.Flatten()(x)
+    x = layers.Dropout(0.5)(x)
+    x = layers.Dense(128, activation='relu')(x)
+    x = layers.Dropout(0.5)(x)
+    x = layers.Dense(_num_classes, activation='softmax')(x)
+    return Model(inputs=_input, outputs=x)
+
+
+def get_dataset(_num_classes):
+    # Load the data and split it between train and test sets
+    (x_train, y_train), (x_test, y_test) = datasets.mnist.load_data()
+
+    # Normalize images
+    x_train = x_train.astype("float32") / 255
+    x_test = x_test.astype("float32") / 255
+
+    # Add Channels axis to data
+    x_train = np.expand_dims(x_train, -1)
+    x_test = np.expand_dims(x_test, -1)
+
+    # convert class vectors to binary class matrices
+    y_train = tf.keras.utils.to_categorical(y_train, _num_classes)
+    y_test = tf.keras.utils.to_categorical(y_test, _num_classes)
+    return (x_train, y_train), (x_test, y_test)
+
+
+def gen_representative_dataset(_images):
+    def _generator():
+        for _img in _images:
+            yield [_img[np.newaxis, ...]]
+    return _generator().__next__
+
+
+if __name__ == "__main__":
+    # init parameters
+    num_classes = 10
+    input_shape = (28, 28, 1)
+    batch_size = 128
+    epochs = 15
+
+    # init model
+    model = get_model(num_classes, input_shape)
+    model.summary()
+
+    # init dataset
+    (x_train, y_train), (x_test, y_test) = get_dataset(num_classes)
+
+    # train float model
+    model.compile(loss="categorical_crossentropy", optimizer="adam", metrics=["accuracy"])
+    model.fit(x_train, y_train, batch_size=batch_size, epochs=epochs, validation_split=0.2)
+
+    # evaluate float model
+    score = model.evaluate(x_test, y_test, verbose=0)
+    print(f"Float model test accuracy: {score[1]:02.4f}")
+
+    # prepare model for QAT with MCT and return to user for fine-tuning
+    representative_dataset = gen_representative_dataset(x_train)
+
+    qat_model, _, _ = mct.keras_quantization_aware_training_init(model,
+                                                                 representative_dataset,
+                                                                 core_config=mct.CoreConfig(n_iter=10),
+                                                                 target_platform_capabilities=get_tpc())
```

Comment:
Please add a comment on why we use here a different TPC and how to obtain HW-specific TPC.